package.json

```diff
@@ -1,55 +1,63 @@
 {
-  "name": "emailjs",
-  "description": "send text/html emails and attachments (files, streams and strings) from node.js to any smtp server",
-  "version": "2.2.0",
-  "author": "eleith",
-  "contributors": [
-    "izuzak",
-    "Hiverness",
-    "mscdex",
-    "jimmybergman",
-    "zackschuster"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "http://github.com/eleith/emailjs.git"
-  },
-  "dependencies": {
-    "addressparser": "^0.3.2",
-    "emailjs-mime-codec": "^2.0.7"
-  },
-  "devDependencies": {
-    "chai": "^4.1.2",
-    "eslint": "^5.1.0",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-mocha": "^5.1.0",
-    "eslint-plugin-prettier": "^2.6.2",
-    "mailparser": "^2.2.0",
-    "mocha": "^5.2.0",
-    "prettier": "^1.13.7",
-    "rollup": "^0.62.0",
-    "rollup-plugin-commonjs": "^9.1.3",
-    "rollup-plugin-node-resolve": "^3.3.0",
-    "smtp-server": "^3.4.6"
-  },
-  "engine": [
-    "node >= 6"
-  ],
-  "main": "email.js",
-  "scripts": {
-		"rollup": "rollup -c rollup.config.js && npm run rollup:test",
-		"rollup:test": "npm run test -- --file rollup/email.bundle.test.js",
-    "test": "mocha"
-  },
-  "license": "MIT",
-  "eslintIgnore": [
-    "rollup.config.js",
-    "rollup/email.bundle.js",
-    "email.esm.js"
-  ],
-  "prettier": {
-    "singleQuote": true,
-    "trailingComma": "es5",
-    "useTabs": true
-  }
+	"name": "emailjs",
+	"description": "send text/html emails and attachments (files, streams and strings) from node.js to any smtp server",
+	"version": "2.2.0",
```

Comment:
given the sheer churn involved with this PR, i'm strongly for semver-major

Reply:
re: the API changes — the ESM export should to me still be considered experimental, since we can't test it until userland catches up to the implementation. there might in fact be serious bugs with the code that we just won't see until we have a story for running it against tests.

i'm also worried i didn't get the conditional exports correct. [according to node's docs](https://nodejs.org/api/esm.html#esm_conditional_exports) we should be good, though i did just now add the `"type": "module"`flag to be sure (i had taken it out before when i thought it might have something to do with testing breaking).

Reply:
let's bump to 3.0

Reply:
i tested conditional exports with `yarn link` & everything resolved correctly. i also diffed the bundles & they're practically identical. based on all this i think the ESM export can be considered "stable".